CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 ### Removed
 
+## [7.0.1]
+### Changed
+- Updated to [Sophia 7.0.1](https://github.com/aeternity/aesophia/blob/master/CHANGELOG.md#701)
+- Rename APIVersion to ApiVersion
+- Don't require options object
+
 ## [7.0.0]
 ### Changed
 - Updated to [Sophia 7.0.0](https://github.com/aeternity/aesophia/blob/master/CHANGELOG.md#700---2021-10-20)
```

Comment:
Same problem here as in cli 🙈

Reply:
Also, these changes were in 7.0.0 not in 7.0.1?

Reply:
@hanssv Which changes do you mean?

This 2 changes were merged after the release of `v7.0.0` on the 4th of August:

- Rename APIVersion to ApiVersion
- Don't require options object

They were not part of `v7.0.0`.

Reply:
The broken link was fixed here 216bbdb7a24e0f35e131027aae06694a018a370d